wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxDataSource.swift

```diff
@@ -95,18 +95,17 @@ class MessageToolboxDataSource {
      * - parameter widthConstraint: The width available to rend the toolbox contents.
      */
 
-    func updateContent(forceShowTimestamp: Bool, widthConstraint: CGFloat) -> SlideDirection? {
+    func shouldUpdateContent(widthConstraint: CGFloat) -> Bool {
```

Comment:
Don't forget to update the doc comment (still shows `forceShowTimestamp` parameter).

Reply:
Looking more at this method, I don't think the name is quite right. `shouldUpdateContent()` suggests that we want to compute whether we should update the content. But actually we do potentially update the content, and return true if we did. It would make sense to leave the name as it was, but add a doc comment explaining what the returned bool means.

Reply:
Done!